autogen/cache/cosmos_db_cache.py

```diff
@@ -0,0 +1,94 @@
+pip install azure-cosmos
+
+import pickle
+from typing import Any, Optional, Union
+from azure.cosmos import CosmosClient, PartitionKey
+from .abstract_cache_base import AbstractCache
```

Comment:
Try and avoid mixing relative and absolute imports. Just stick to absolute imports